wiki/en/en-Tips-Tricks-More.md

```diff
@@ -53,11 +53,20 @@ function loadserverstat ( $statfilename )
 
 You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Do this with "File > Save Mixer Channels Setup" in your client and load them using "Load Mixer Channels Setup" (or drag/drop them to the mixer window).
 
+## Delegate mixing to a mix master using the --mastermix server mode
+
+By starting a server instance with the `--mastermix` flag, the first client to connect will become the mix master.
+
+Subsequent joiners will be able to control their own as well as the mix master's pan and gain, but nobody else's (clients after version TODO will see their faders and buttons grayed out). Instead, they will hear the pan and gain value as set by the mix master.
+
+When this mix master sets a channel to SOLO, this will *not* apply to everybody's elses mix. Allowing the mix master to 'secretly' SOLO people can help creating a good mix without disturbing other clients.
```

Comment:
House style has been to refer to buttons in the UI with sentence case, so this would be "sets a channel to Solo" and "'secretly' solo" (the latter because its just a verb not a button that's being referred to). 

Meta: This is a good example of the use of "channel" and "people" as per the debate about when to use "channel". I think, but can't remember and now can't find the discussion on this, but were we saying that we should use "channel" only to refer to system audio (eg "the server mixes all audio channels", "channel panning"), otherwise everyone is a "person" unless you want to refer to the UI, in which case it's a "fader" (eg "the icon above a person's fader")?

Reply:
The problem with "person" is a "person" can be playing several instruments and run multiple clients, each with the same name.  Whenever we need to refer to identifying controls on the mixer - i.e. when referring to actions that would analogously be undertaken on a physical mixing desk - then "channels" should be used, in my view.

Reply:
> The problem with "person" is a "person" can be playing several instruments and run multiple clients,

Yes, I think it was for reasons like this that we wanted to use "person", so that it could use as opposed to "channel", for clarity. So for example, "You can group a person's channels..." and "People connected to the server". If we didn't have "person" and always used "channel" then it would be less clear in some contexts perhaps.

>  when referring to actions that would analogously be undertaken on a physical mixing desk

Although I recall we decided that we'd use "fader" in that case, eg "Lower the channel's fader" not "Lower the volume on the channel", reserving "channel" for the "system thing" if we were talking about something from the user's POV.

Perhaps bit of a subtle distinction, but it was in an effort to have three distinct terms for a human using a server ("person"), a UI element ("fader"), and the system thing ("channel") so as make things a bit clearer.

So this would mean we'd say eg, "Muting a person's channel will display a muted icon above your fader" NOT "Muting a channel will display a muted icon above your channel".